seedpod_ground_risk/ui_resources/info_popups.py

```diff
@@ -0,0 +1,92 @@
+import matplotlib.pyplot as mpl
+from PySide2.QtWidgets import QDialog, QVBoxLayout
+from matplotlib import gridspec
+from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT as NavigationToolbar
+
+from seedpod_ground_risk.pathfinding.moo_ga import *
```

Comment:
Why is this imported?

Reply:
Imported to allow for numpy and 'line' function to be used.